tools/wptrunner/docs/expectation.rst

```diff
@@ -305,6 +230,137 @@ and less than or equal to 200 total pixels different. For any
 comparison involving ref1.html on the right hand side, the limits
 would instead be a difference per channel not more than 20 and a total
 difference count of not less than 200 and not more than 300. For the
-specific comparison subtest1.html == ref2.html (both resolved against
+specific comparison ``subtest1.html == ref2.html`` (both resolved against
 the test URL) these limits would instead be 10 to 15 and 0 to 20,
 respectively.
+
+Generating Expectation Files
+----------------------------
+
+wpt provides the tool ``wpt update-expectations`` command to generate
+expectation files from the results of a set of test runs. The basic
+syntax for this is::
+
+  ./wpt update-expectations [options] [logfile]...
+
+Each ``logfile`` is a wptreport log file from a previous run. These
+can be generated from wptrunner using the ``--log-wptreport`` option
+e.g. ``--log-wptreport=wptreport.json``.
```

Comment:
Do we want to make allowances for `--log-raw` etc., or just only document the `--log-wptreport` case?

Reply:
Let's just document the `--log-wptreport` case; `--log-raw` adds quite a lot of unnecessary parsing overhead so it shouldn't be used in most cases.

Reply:
Does it make sense to throw an error when we have any other format? IIRC, some of the confusion historically has been providing the wrong format (like TBPL) appears to silently work?

Reply:
Probably, but that's more or less unrelated to this PR.